src/components/screens/wallet/overview/index.js

```diff
@@ -131,15 +131,10 @@ const Overview = ({
         <BalanceInfo
           t={t}
           activeToken={activeToken}
-          balance={balance}
```

Comment:
why change this way for sending required props? maybe we should stick what it was rather than changing? could you please explain the reasoning behind this change

Reply:
@ManuGowda Balance, pomStart, address and isBanned are all gotten from the account object, so it makes sense that since we are passing the account object, they can all be gotten from it.